components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -1254,13 +1279,19 @@ - (void)stopForwardingTouchEventsForView:(UIView *)view {
 }
 
 - (void)setMinimumHeight:(CGFloat)minimumHeight {
-  _hasExplicitlySetMinHeight = YES;
+  _hasExplicitlySetMinOrMaxHeight = YES;
   if (_minimumHeight == minimumHeight) {
     return;
   }
 
   _minimumHeight = minimumHeight;
 
+  if (_minMaxHeightIncludesSafeArea) {
+    _minHeightIncludingSafeArea = _minimumHeight;
```

Comment:
Might be able to save a few characters by using `With` instead of `Including`.

Reply:
Good idea, might do that in a follow up PR as I don't want to lose my position in the Travis queue 😞

Reply:
Np - 🚢it!